src/php/fpm/conf/zz-docker.conf.template

```diff
@@ -2,6 +2,69 @@
+;             following directives. With this process management, there will be
+;             always at least 1 children.
+;             pm.max_children      - the maximum number of children that can
+;                                    be alive at the same time.
+;             pm.start_servers     - the number of children created on startup.
+;             pm.min_spare_servers - the minimum number of children in 'idle'
+;                                    state (waiting to process). If the number
+;                                    of 'idle' processes is less than this
+;                                    number then some children will be created.
+;             pm.max_spare_servers - the maximum number of children in 'idle'
+;                                    state (waiting to process). If the number
+;                                    of 'idle' processes is greater than this
+;                                    number then some children will be killed.
+;  ondemand - no children are created at startup. Children will be forked when
+;             new requests will connect. The following parameter are used:
+;             pm.max_children           - the maximum number of children that
+;                                         can be alive at the same time.
+;             pm.process_idle_timeout   - The number of seconds after which
+;                                         an idle process will be killed.
+; Note: This value is mandatory.
+pm = ${PHP_FPM_PM}
+
+; The number of child processes to be created when pm is set to 'static' and the
+; maximum number of child processes when pm is set to 'dynamic' or 'ondemand'.
+; This value sets the limit on the number of simultaneous requests that will be
+; served. Equivalent to the ApacheMaxClients directive with mpm_prefork.
+; Equivalent to the PHP_FCGI_CHILDREN environment variable in the original PHP
+; CGI. The below defaults are based on a server without much resources. Don't
+; forget to tweak pm.* to fit your needs.
+; Note: Used when pm is set to 'static', 'dynamic' or 'ondemand'
+; Note: This value is mandatory.
+pm.max_children = ${PHP_FPM_PM_MAX_CHILDREN}
+
+; The number of child processes created on startup.
+; Note: Used only when pm is set to 'dynamic'
+; Default Value: min_spare_servers + (max_spare_servers - min_spare_servers) / 2
+pm.start_servers = ${PHP_FPM_PM_START_SERVERS}
+
+; The desired minimum number of idle server processes.
+; Note: Used only when pm is set to 'dynamic'
+; Note: Mandatory when pm is set to 'dynamic'
+pm.min_spare_servers = ${PHP_FPM_PM_MIN_SPARE_SERVERS}
+
+; The desired maximum number of idle server processes.
+; Note: Used only when pm is set to 'dynamic'
+; Note: Mandatory when pm is set to 'dynamic'
+pm.max_spare_servers = ${PHP_FPM_PM_MAX_SPARE_SERVERS}
+
+; The number of seconds after which an idle process will be killed.
+; Note: Used only when pm is set to 'ondemand'
+; Default Value: 10s
```

Comment:
This says `10s` but the environment var only has `10` (without `s`)

Reply:
actually it's reported as `10` by php-fpm:
```
docker run -it --rm php:7.3-fpm-alpine3.9 php-fpm -tt
...
[23-Sep-2019 15:50:05] NOTICE: 	pm = dynamic
[23-Sep-2019 15:50:05] NOTICE: 	pm.max_children = 5
[23-Sep-2019 15:50:05] NOTICE: 	pm.start_servers = 2
[23-Sep-2019 15:50:05] NOTICE: 	pm.min_spare_servers = 1
[23-Sep-2019 15:50:05] NOTICE: 	pm.max_spare_servers = 3
[23-Sep-2019 15:50:05] NOTICE: 	pm.process_idle_timeout = 10
[23-Sep-2019 15:50:05] NOTICE: 	pm.max_requests = 0

```

Reply:
Ok, great! Didn't know that, thanks